JellyfinPlayer tvOS/VideoPlayer/MediaInfoView.swift

```diff
@@ -0,0 +1,117 @@
+//
+/*
+ * SwiftFin is subject to the terms of the Mozilla Public
+ * License, v2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2021 Aiden Vigue & Jellyfin Contributors
+ */
+
+import SwiftUI
+import JellyfinAPI
+
+class MediaInfoViewController: UIViewController {
+    private var contentView: UIHostingController<MediaInfoView>!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tabBarItem.title = "Info"
+    }
+    
+    func setMedia(item: BaseItemDto)
+    {
+        contentView = UIHostingController(rootView: MediaInfoView(item: item))
+        self.view.addSubview(contentView.view)
+        contentView.view.translatesAutoresizingMaskIntoConstraints = false
+        contentView.view.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+        contentView.view.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+        contentView.view.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        contentView.view.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+        
+    }
+}
+
+struct MediaInfoView: View {
+    @State var item : BaseItemDto? = nil
+    
+    var body: some View {
+        if let item = item {
+            HStack {
+                VStack {
+                    ImageView(src: item.type == "Episode" ? item.getSeriesPrimaryImage(maxWidth: 200) : item.getPrimaryImage(maxWidth: 200), bh: item.type == "Episode" ? item.getSeriesPrimaryImageBlurHash() : item.getPrimaryImageBlurHash())                    .frame(width: 200, height: 300)
+                        .cornerRadius(10)
+                    Spacer()
+                }
+                .padding(.leading, 200)
```

Comment:
This line makes the dialog look off-center in my opinion.

Reply:
I've aimed at replicating the native player here:
![Panel info tab](https://user-images.githubusercontent.com/47413006/122880331-3d6f2900-d37d-11eb-92e0-510c0ce40c3e.png)

Maybe the slight differences from mine overall doesn't look as good and might be better to center it more?

Reply:
It looks like the content is centered and as small as possible. The dialog in the above image changes size horizontally to match the screen rather than wrapping the inner content exactly, which is why you're running into this issue.